### PR TITLE
[MNOE-1204] licenses n/a present on confirm page not on subscriptions page

### PR DESCRIPTION
**Pull request by jin521**

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -748,10 +748,11 @@
   "mno_enterprise.templates.dashboard.provisioning.subscription.back": "back",
   "mno_enterprise.templates.dashboard.provisioning.subscription.info": "Info",
   "mno_enterprise.templates.dashboard.provisioning.subscription.subscription_info": "Subscription info",
+  "mno_enterprise.templates.dashboard.provisioning.subscription.not_available": "N/A",
```
Comment (ouranos) [inline src/locales/en.json]: If you update the locales they need to be updated in `mno-enterprise` too

File: src/app/views/provisioning/subscriptions.html [hunk 2]
```diff
@@ -99,7 +99,15 @@ <h2 class="section-title" ng-hide="vm.cartSubscriptions" translate>
           <span class="cell-label visible-xs-inline">
             {{ 'mno_enterprise.templates.dashboard.provisioning.subscriptions.licenses' | translate }}:
           </span>
-          <span title="{{subscription.max_licenses}}">{{subscription.max_licenses}}</span>
+          <div ng-show="vm.subscription.max_licenses">
+            {{ vm.subscription.max_licenses }}
+          </div>
+          <div ng-hide="vm.subscription.max_licenses">
+            {{ 'mno_enterprise.templates.dashboard.provisioning.confirm.no_licenses.message' | translate }}
```
Comment (ouranos) [inline src/app/views/provisioning/subscriptions.html]: This shouldn't use this locale namespace "mno_enterprise.templates.dashboard.provisioning.confirm" but "mno_enterprise.templates.dashboard.provisioning.subscriptions".
Even if that create some duplication in the locales, this allows us to precisely control the text.

File: src/app/views/provisioning/subscriptions.html [hunk 3]
```diff
@@ -99,7 +99,15 @@ <h2 class="section-title" ng-hide="vm.cartSubscriptions" translate>
...
+            <span tooltip-append-to-body="true" uib-tooltip="{{ 'mno_enterprise.templates.dashboard.provisioning.confirm.no_licenses.tooltip' | translate}}">
```
Comment (ouranos) [inline src/app/views/provisioning/subscriptions.html]: Same here

**Comment by adamaziz15** [comment]
Lgtm, other than the locale namespace Oli pointed out in subscription.html.

**Pull request merged**